sphinx/source/docs/user_guide/compat.rst

```diff
@@ -9,32 +9,28 @@ you to use the most appropriate tool for each task.
 BokehJS
 -------
 
-Bokeh generates JavaScript, and so Bokeh output can be combined with a
-wide variety of JavaScript libraries, such as `React`_. Listing
-such libraries is beyond the scope of this document; it's best just to
+Bokeh generates JavaScript. This makes its output fully compatible with
+a wide variety of JavaScript libraries, such as `React`_. Listing all
+such libraries is beyond the scope of this document; it's best to just
```

Comment:
I very much agree with @Konntroll's comment on this paragraph ("Perhaps it's better to offer a list of libraries that are commonly used and let the user know that there's more synergy beyond the list. As things are, this has a rather uninformative feel to it."). @bryevdv, do you have an idea on how to be more concrete here?

Reply:
Well, "Bokeh generates JavaScript" is slightly misleading. Bokeh generates JSON that is consumed by an existing (not generated) JS library, BokehJS. Maybe something like:

> All of the actual work to display Bokeh content in a browser is done by a companion JavaScript library, BokehJS. Normally, BokehJS operates by rendering JSON data that is generated by the Python Bokeh library. However,  BokehJS can also be used directly as JavasScript library on its own, or together with common JavaScript frameworks such as `React`_, `Vue`_, `Angular`_, and others.

Reply:
The user guide has a whole chapter on BokehJS (https://docs.bokeh.org/en/dev/docs/user_guide/bokehjs.html) and I suggest linking to that section instead of providing too many details here. How about:

> To display visualizations in a browser, Bokeh uses its companion JavaScript library, [BokehJS](https://docs.bokeh.org/en/dev/docs/user_guide/bokehjs.html). You can also use BokehJS as a standalone JavasScript library or in combination with JavaScript frameworks such as React_, Vue_, Angular_, and others. 
See [Developing with JavaScript](https://docs.bokeh.org/en/dev/docs/user_guide/bokehjs.html) for more information about BokehJS.

Reply:
Sounds good to me, but maybe mention JSON ("Bokeh uses its companion JavaScript  library, BokehJS, communicating data and events from Python using JSON")?